### DISCUSSION TITLE
Socket exception while calling create data base operation.

### DISCUSSION THREAD

**Discussion by ShekharReddyN** [Azure/azure-cosmos-dotnet-v3#3104] [General]
I am getting the socket exception while calling CreateDatabaseIfNotExistsAsync 

![image](https://user-images.githubusercontent.com/6689368/159446738-76326624-a5cf-4351-9182-7e909f1cf4a4.png)

Using the V3 - 3.26.1 version SDK.

Could you please help me resolve this issue?

**Comment by FabianMeiswinkel** [comment]
Seems to be a DNS resolution issue on your environment - see https://docs.microsoft.com/en-us/windows/win32/winsock/windows-sockets-error-codes-2?redirectedfrom=MSDN#WSANO_DATA

**Reply by ShekharReddyN** [reply]
Thanks. Checking on this.
